### DISCUSSION TITLE
I went through the rabbit hole of PKGBUILD

### DISCUSSION THREAD

**Discussion by Kiaos** [LizardByte/community#198] [Sunshine] [answered]
So I spent about an hour installing dependncy after dependincy to get the PKGBUILD and Finally got it working but i needed to type sunshine in konsole and then setup pin. 

Now that its working the lag is horendous. I had to install a few x11 things. No i mean a shit ton of x11 things.
Im not sure its using wayland. 

So yea. not sure I know what I did wrong. At one point in the PKGBUILD it said it didnt see wayland.

I just followed the errors.

Btw I never got the flatpacks working.

BTW im on a steam deck so im not sure if that is the reason its slow.

**Comment by ReenigneArcher** [comment, marked as answer]
You need to setcap on the sunshine binary to use Wayland. This is in our docs.
